src/locales/da.json

```diff
@@ -9,8 +9,7 @@
     },
     "404": {
       "title": "404 Siden findes ikke",
-      "subtext_html":
-        "Siden du leder efter findes ikke. Klik <a href=\"/collections/all\">her</a> for at fortsætte med at handle."
+      "subtext_html": "Siden du leder efter findes ikke. Klik <a href=\"/collections/all\">her</a> for at fortsætte med at handle."
```

Comment:
These are style changes -- is this your prettier config? Should the project have a Prettier config so these changes do revert? Should this change even be happening?